foreman/foreman.spec

```diff
@@ -46,8 +50,11 @@ Requires: wget
 Requires: /etc/cron.d
 Requires(pre):  shadow-utils
 Requires(post): chkconfig
+Requires(post): systemd-sysv
```

Comment:
I recall there being a macro for the requirements as well

Reply:
Sorry, what macro/where? I'm missing something here

Reply:
https://github.com/theforeman/foreman-packaging/blob/dc21dfd9cc15499785e4da30cdad67131d99e52a/foreman/foreman.spec#L132

Reply:
That macro adds systemd requires, not systemd-sysv or systemd-units (https://github.com/systemd/systemd/blob/master/src/core/macros.systemd.in)

Reply:
Why are those needed? I can't even find systemd-units on my Fedora or on a CentOS 7 machine.

Reply:
To answer my own question: the ```systemd``` RPM has a provides ```systemd-units``` so I'd prefer to use the requires macro. We should also look at the ```chkconfig``` and ```initscripts``` requirements but that might be out of scope for this PR.

Reply:
systemd-sysv is meant for automatic compatibility with SysV

Reply:
Ok, I'd prefer to follow the Fedora guidelines here and this is fine then.